doc, installer: win - new logo in installer; optim .jpg

eljefedelrodeodeljefe opened the pull request:
### Pull Request check-list

_Please make sure to review and check all of these items:_

- [x] Is the commit message formatted according to [CONTRIBUTING.md][0]?

_NOTE: these things are not required to open a PR and can be done
afterwards / while the PR is open._

### Affected core subsystem(s)

`tools`

[0]: https://github.com/nodejs/node/blob/master/CONTRIBUTING.md#step-3-commit

### Description of change

Latest logo iteration makes image adaptation necessary, #5381.
I couldn't get wix or VS to run properly (new to windows dev), but according to what I see in the installer and read in the wix project, this should suffice.

had to close #5391 and reopen here due to rebase foo and GH not being able to reopen closed force-pushed branches. Sorry.

/cc @rvagg can you give Jenkins a try? Could build locally.

rvagg commented:
https://nodejs.org/download/test/v6.0.0-test20160303761abde90c/

I don't have a Windows machine or VM handy right now to test so could you screenshot both the x86 and x64 installers and post them here please @eljefedelrodeodeljefe?

eljefedelrodeodeljefe commented:
On parallels, Win10:

**x64**
<img width="406" alt="node_js_setup" src="https://cloud.githubusercontent.com/assets/3899684/13489626/6673998c-e128-11e5-8351-d47ba6802355.png">
<img width="418" alt="node_js_setup" src="https://cloud.githubusercontent.com/assets/3899684/13489669/9fc804d4-e128-11e5-97c1-0f986288789c.png">

**x86**
<img width="428" alt="node_js_setup" src="https://cloud.githubusercontent.com/assets/3899684/13489709/d51cf45a-e128-11e5-91f2-a00c8b540902.png">
<img width="413" alt="node_js_setup" src="https://cloud.githubusercontent.com/assets/3899684/13489727/f47cd4a0-e128-11e5-8cba-da3fede2581b.png">

rvagg commented:
Nice, my only question now is whether we should change the Node green bar at the top to match the new style. @nodejs/website can you please look at the above screenshots and tell us if they are acceptable or should we move the strip of green at the top to a different colour, and if so, what colour?

joaocgreis commented:
Tested both installers on Windows 2008 R2, both look fine, as the screenshots above for Windows 10.

eljefedelrodeodeljefe commented:
Changed the color to the middle tone. See RGB in commit. Squashed the thing. 

@rvagg When you decide to land this, I'd actually propose on a separate issue to redesign the installer to something more win10 "tile"-like design. Something that might be nice on my quest for a better win dev UX. Maybe then someone wants to throw in some design ideas.

Fishrock123 commented:
Could you make sure to run the image though imageoptim or similar tools? (Make sure you don't loose quality though)

eljefedelrodeodeljefe commented:
ran it through imageoptim. Do you see anything specific?

Fishrock123 commented:
No I just wanted to make sure, less binary bytes in git.

eljefedelrodeodeljefe commented:
Ah, alright. FYI It's 7 and 9kb now, with 9-14% optimization. When I did it with `.png` it was even over 30%.

ChALkeR commented:
@Fishrock123 Btw, `optipng` does not give the best result. Even running plain simple `advdef` afterwards saves a few more bytes. Note that `advdef` is not even a png optimizer, it does not touch the internal png structure. It just recompresses things.

silverwind commented:
LGTM

rvagg commented:
lgtm

landed @ 44c9751 thanks @eljefedelrodeodeljefe
